mappings/net/minecraft/tag/Tag.mapping

```diff
@@ -14,9 +14,55 @@ CLASS net/minecraft/class_3494 net/minecraft/tag/Tag
 		ARG 1 entry
 	METHOD method_15142 getRandom (Ljava/util/Random;)Ljava/lang/Object;
 		ARG 1 random
+	METHOD method_26777 of (Ljava/util/Set;)Lnet/minecraft/class_3494;
+		ARG 0 set
 	CLASS class_3495 Builder
 		COMMENT A builder class to ease the creation of tags. It can also be used as a
 		COMMENT mutable form of a tag.
+		FIELD field_23688 entries Ljava/util/Set;
+		METHOD method_26778 create ()Lnet/minecraft/class_3494$class_3495;
+		METHOD method_26779 add (Lnet/minecraft/class_3494$class_3496;)Lnet/minecraft/class_3494$class_3495;
+			ARG 1 entry
+		METHOD method_26780 read (Lcom/google/gson/JsonObject;)Lnet/minecraft/class_3494$class_3495;
+			ARG 1 json
+		METHOD method_26782 build (Ljava/util/function/Function;Ljava/util/function/Function;)Ljava/util/Optional;
+			ARG 1 tagGetter
+			ARG 2 objectGetter
+		METHOD method_26784 add (Lnet/minecraft/class_2960;)Lnet/minecraft/class_3494$class_3495;
+			ARG 1 id
+		METHOD method_26785 streamEntries ()Ljava/util/stream/Stream;
+		METHOD method_26786 streamUnresolvedEntries (Ljava/util/function/Function;Ljava/util/function/Function;)Ljava/util/stream/Stream;
```

Comment:
```suggestion
		METHOD method_26786 unresolvedEntryStream (Ljava/util/function/Function;Ljava/util/function/Function;)Ljava/util/stream/Stream;
```

Reply:
Same, streaming unresolved entries

Reply:
Also prefer liach's here for same reason as above